doc-content/enterprise-only/installation/clustering-download-eap-ps-proc.adoc

```diff
@@ -1,28 +1,55 @@
 [id='clustering-download-eap-ps-proc']
-= Downloading and extracting {EAP} 7.1 and {KIE_SERVER}
-Complete the steps in this section to download and install {EAP} 7.1 and download and repackage {KIE_SERVER} for installation in a clustered environment.
+= Downloading and extracting {EAP} 7.2 and {KIE_SERVER}
+Complete the steps in this section to download and install {EAP} 7.2 and {KIE_SERVER} for installation in a clustered environment.
 
 .Procedure
-. Install {EAP} 7.1 on each node of the cluster:
+. Download {EAP} 7.2 on each node of the cluster:
```

Comment:
isn't it possible to use a variable for versioning?

Reply:
It is.

Reply:
I have done this  globally in another branch so the change won't appear in this branch.